src/XRP/network-clients/web-socket-network-client.ts

```diff
@@ -275,6 +279,40 @@ export default class WebSocketNetworkClient {
     return accountOffersResponse as AccountOffersResponse
   }
 
+  /**
+   * Submits a ripple_path_find request to the rippled WebSocket API.
+   * @see https://xrpl.org/ripple_path_find.html
+   *
+   * @param sourceAccount
+   */
+  public async findRipplePath(
+    sourceAccount: string,
+    destinationAccount: string,
+    destinationAmount: string | IssuedCurrency,
+    sendMax?: string | IssuedCurrency,
+    sourceCurrencies?: SourceCurrency[],
+  ): Promise<RipplePathFindResponse> {
+    const ripplePathFindRequest: RipplePathFindRequest = {
+      id: `${RippledMethod.ripplePathFind}_${sourceAccount}_${this.idNumber}`,
+      command: RippledMethod.ripplePathFind,
+      source_account: sourceAccount,
```

Comment:
does this assume that sourceAccount and destinationAccount are encoded as classic addresses?  we might want to include that in the documentation, or add a check where if they're X-addresses, they get decoded before being submitted